algorithms/Python/linked_lists/circularLinkedList.py

```diff
@@ -0,0 +1,130 @@
+"""
+Data Structure : Circular Linked List.
+Time Complexity : O(1) for all operations.
+Space Complexity : O(n) Where, 'n' is the number of nodes in the linked list.
+"""
+
+
+class Node:
+    """
+    Class to represent a Node.
+    Attributes:
+        - value (int) : The value of the node.
+        - next (Node) : The reference of the next node.
+    """
+
+    def __init__(self, value: int):
+        self.value = value
+        self.next = None
```

Comment:
Adding a type hint would help mypy to understand the actual data type of the variable and would remove the error caused on line 75 and 85.

```suggestion
        self.next:Optional[Node] = None
```

Reply:
Thank you for your help @Arsenic-ATG